tests/frontend/confirm_controller_test.js

```diff
@@ -42,41 +42,129 @@ describe("Confirm controller", () => {
     application.register("confirm", ConfirmController);
   });
 
-  describe("initial state", function() {
-    describe("the button", function() {
-      it("is disabled", function() {
+  describe("initial state", function () {
+    describe("the button", function () {
+      it("is disabled", function () {
```

Comment:
yeah awesome point, it came after I ran `prettier` on it locally (muscle memory in VS Code) 😞 It may make sense to have a style plugin for stuff like this if folks want it (maybe a local Prettier config), but the initial formats to these files will cause a lot of diff noise like this